modules/ROOT/pages/installation/linux/rpm.adoc

```diff
@@ -21,18 +21,24 @@ However, Java supports pointing to an alternative `temp` directory by either set
 +
 Use the following as `root` to add the repository:
 +
-[source, shell]
+[source, shell, subs="attributes"]
 ----
 rpm --import https://debian.neo4j.com/neotechnology.gpg.key
 cat <<EOF >  /etc/yum.repos.d/neo4j.repo
 [neo4j]
 name=Neo4j RPM Repository
-baseurl=https://yum.neo4j.com/stable
+baseurl=https://yum.neo4j.com/stable/{neo4j-version}
 enabled=1
 gpgcheck=1
 EOF
 ----
 +
+[NOTE]
+====
+If you are upgrading from an older version of Neo4j, you may need to clear the package manager cache before neo4j packages become available:
```

Comment:
```suggestion
If you are upgrading from an older version of Neo4j, you may need to clear the package manager cache before Neo4j packages become available:
```

Reply:
I think they might not see this note if they are upgrading because all the upgrade information is in the Upgrade and Migration guide.

Reply:
I actually think that this note should be added to the U&M guide instead of here.